io/azure.go

```diff
@@ -0,0 +1,104 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package io
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/url"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob"
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob/container"
+	"gocloud.dev/blob"
+	"gocloud.dev/blob/azureblob"
+)
+
+// Constants for Azure configuration options
+const (
+	ADLS_SAS_TOKEN_PREFIX         = "adls.sas-token."
+	ADLS_CONNECTION_STRING_PREFIX = "adls.connection-string."
+	ADLS_READ_BLOCK_SIZE          = "adls.read.block-size-bytes"
+	ADLS_WRITE_BLOCK_SIZE         = "adls.write.block-size-bytes"
+	ADLS_SHARED_KEY_ACCOUNT_NAME  = "adls.auth.shared-key.account.name"
+	ADLS_SHARED_KEY_ACCOUNT_KEY   = "adls.auth.shared-key.account.key"
```

Comment:
We don't tend to use the ALL_CAP_SNAKE_CASE for constants, instead we use the format `ADLSSasTokenPrefix` etc. please update

Reply:
updated.